src/main/java/org/opensearch/neuralsearch/plugin/NeuralSearch.java

```diff
@@ -71,4 +76,11 @@ public List<QuerySpec<?>> getQueries() {
             new QuerySpec<>(NeuralQueryBuilder.NAME, NeuralQueryBuilder::new, NeuralQueryBuilder::fromXContent)
         );
     }
+
+    @Override
+    public Map<String, Processor.Factory> getProcessors(Processor.Parameters parameters) {
+        final MachineLearningNodeClient machineLearningNodeClient = new MachineLearningNodeClient(parameters.client);
+        final MLCommonsClientAccessor clientAccessor = new MLCommonsClientAccessor(machineLearningNodeClient);
```

Comment:
This is what was not suggested in the older comment. The idea was to use the instance of  MLCommonsClientAccessor which was created in createComponents Function

Reply:
![Screen Shot 2022-10-14 at 7 41 45 PM](https://user-images.githubusercontent.com/6162415/195965381-05e8c46d-c77a-490e-87ed-49d4a65d8474.jpg)

Reply:
Done